puppet/zulip/manifests/app_frontend_base.pp

```diff
@@ -5,11 +5,15 @@
   include zulip::nginx
   include zulip::supervisor
 
-  $web_packages = [
-    # Needed to access our database
-    "postgresql-client-${zulip::base::postgres_version}",
-  ]
-  zulip::safepackage { $web_packages: ensure => 'installed' }
+  # No need to ensure client tools on redhat, since they're shipped in
+  # postgresql*
```

Comment:
I think this change is incorrect in the case that the user is running the app frontend on a different system than the postgres server.  I'm OK with merging as-is with a mention of that in a comment (that we don't support a separate postgres server on RHEL) and leave that for a follow-up issue, since it may be somewhat messy to fix.